### ISSUE TITLE
Crash in PyCharm when trying to resize a tooltip

### ISSUE THREAD

**Issue by YaLTeR** [swaywm/sway#3131]
* Sway Version: `1.0-beta.1-193-g0451acfb (Nov 15 2018, branch 'master')`
* Debug Log and Backtrace: https://gist.github.com/YaLTeR/f0d57bead046a6030760b2953f2a0b98
* Configuration File: default except $term is changed

1. Open a Python file in IntelliJ PyCharm.
2. Get a documentation tooltip to appear by clicking on a function and pressing Ctrl-Q.
3. Try to $mod-right mouse button the tooltip (as if to resize the window).

**Comment by emersion** [comment]
Can you try https://github.com/swaywm/sway/pull/3132?

**Comment by YaLTeR** [comment]
Yep, the crash is fixed. The tooltip doesn't behave sensibly when you try moving or resizing it this way but that's besides this issue.
